docker-compose.yml

```diff
@@ -1,5 +1,3 @@
-version: "3.7"
-
```

Comment:
New webcam env params should be added in environment section of dagster_pipeline (L77)